lib/Doctrine/ORM/Configuration.php

```diff
@@ -350,13 +350,11 @@ public function addNamedNativeQuery($name, $sql, Query\ResultSetMapping $rsm)
      *
      * @param string $name The name of the query.
      *
-     * @throws ORMException
+     * @return array A tuple with the first element being the SQL string and the second
```

Comment:
The commenting system does not seem to have worked entirely, there is an issue here I think: you don't specify the types of the elements and phpcs is not going to like that.

Reply:
Sorry I don't understand what you mean here.

Reply:
I now know what you mean. I don't fully understand in which cases phpcs requests this.
The other question is, is this syntax supported by IDEs? Especially I don't know how to solve the last of those issues in
`UnitOfWork::getEntityChangeSet`

Reply:
> I don't fully understand in which cases phpcs requests this.

I think it's any time the types of the elements can be guessed. So for instance `mixed[]` is ok, while `array` is not.